shared/folders/files/render.native.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+import React, {Component} from 'react'
+import {ScrollView} from 'react-native'
+import {Box, Text, BackButton, Avatar, Icon} from '../../common-adapters'
+import File from './file/render'
+import {globalStyles, globalColors} from '../../styles/style-guide'
+import {intersperseFn} from '../../util/arrays'
+import type {Props, FileSection} from './render'
+
+export default class Render extends Component<void, Props, void> {
+  _renderSection (section: FileSection) {
+    return (
+      <Box key={section.name} style={{...globalStyles.flexBoxColumn, backgroundColor: backgroundColorThemed[this.props.theme]}}>
+        <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', height: 32}}>
+          <Box key={section.name} style={{...globalStyles.flexBoxRow, marginLeft: 8}}>
+            {section.modifiedMarker && <Icon type='thunderbolt' style={{height: 12, alignSelf: 'center', marginRight: 6, ...styleSectionTextThemed[this.props.theme]}} />}
+            <Text type='BodySmallSemibold' style={{...styleSectionTextThemed[this.props.theme]}}>{section.name}</Text>
+          </Box>
+        </Box>
+        {intersperseFn(i => <Box key={i} style={{height: 0.5, backgroundColor: globalColors.black_10}} />,
+                       section.files.map(f => <File key={f.name} {...f} />))}
+      </Box>
+    )
+  }
+
+  // TODO render checkerboard pattern for private mode
```

Comment:
Yeah I saw it and spend about 30 min trying to get it working and failed. So I deferred